recognition/advanced_tree_search.py

```diff
@@ -286,18 +272,46 @@ def create_config(
         lmgc_mem: float,
         lmgc_alias: Optional[str],
         lmgc_scorer: Optional[rasr.FeatureScorer],
+        separate_lm_image_gc_generation: bool,
         model_combination_config: Optional[rasr.RasrConfig],
         model_combination_post_config: Optional[rasr.RasrConfig],
         extra_config: Optional[rasr.RasrConfig],
         extra_post_config: Optional[rasr.RasrConfig],
         **kwargs,
     ):
-        lm_gc = AdvancedTreeSearchLmImageAndGlobalCacheJob(
-            crp, lmgc_scorer if lmgc_scorer is not None else feature_scorer, extra_config, extra_post_config
-        )
-        if lmgc_alias is not None:
-            lm_gc.add_alias(lmgc_alias)
-        lm_gc.rqmt["mem"] = lmgc_mem
+        def add_lm_config_to_crp(crp, lm_config):
+            crp = copy.deepcopy(crp)
+            crp.language_model_config = lm_config
+            return crp
+
+        if separate_lm_image_gc_generation:
+            gc_job = BuildGlobalCacheJob(crp, extra_config, extra_post_config)
+
+            arpa_lms = lm.find_arpa_lms(crp.language_model_config, None)
+            lm_image_jobs = {
+                (i + 1): lm.CreateLmImageJob(
+                    add_lm_config_to_crp(crp, lm_config), extra_config=extra_config, extra_post_config=extra_post_config
+                )
+                for i, (lm_config, _lm_post_config) in enumerate(arpa_lms)
```

Comment:
Is there a reason for the `_`. Maybe I am overlooking something in the web view. If its not used you could just fully replace it with `_`

Reply:
I think the `post_config here is unused, because 
* both `config` and `post_confic` are extracted from the `crp`
* only the `config` is modified
* the `crp` with the old `post_config` is still being used

+1 to can be just `_` to make clear it is unused
```suggestion
                for i, (lm_config, _) in enumerate(arpa_lms)
```

Reply:
I think the `post_config here is unused, because 
* both `config` and `post_confic` are extracted from the `crp`
* only the `config` is modified
* the `crp` with the old `post_config` is still being used

+1 to can be just `_` to make clear it is unused
```suggestion
                for i, (lm_config, _) in enumerate(arpa_lms)
```